bindings/kitsune_client/src/op_store.rs

```diff
@@ -287,6 +287,21 @@ impl OpStore for WtOpStore {
         })
     }
 
+    /// Filter the passed in op ids by ops that exist in the store.
```

Comment:
Upgrading K2 should probably have been a separate PR

Reply:
shall I break it out?